platform/server/elasticsearch/Cluster.ts

```diff
@@ -32,41 +31,17 @@ export class Cluster {
     this.log.info('cluster client stopped');
   }
 
-  // Attempt at typing the current `callWithRequest`, which is really difficult
-  // to type properly the way it is right now.
-  // TODO This should _return_ a client preset with the headers, but this
-  // requires changing Elasticsearch.js
-  async withRequest<T>(
-    req: KibanaRequest,
-    cb: (client: Client, headers: { [key: string]: any }) => T,
-    options: {
-      wrap401Errors: boolean;
-    } = {
-      wrap401Errors: false
-    }
-  ) {
-    const headers = req.getFilteredHeaders(this.config.requestHeadersWhitelist);
-
-    try {
-      return await cb(this.client, headers);
-    } catch (err) {
-      // instanceof err check?
-
-      if (options.wrap401Errors && err.statusCode === 401) {
-        console.log('401 + wrap', err);
-        throw err;
-
-        // From current Kibana:
+  callWithRequest(endpoint: string, clientParams = {}, options = {}): any {
+    return {};
+    //if (req.headers) {
+    //  const filteredHeaders = filterHeaders(req.headers, this.getRequestHeadersWhitelist());
+    //  set(clientParams, 'headers', filteredHeaders);
+    //}
 
-        // const boomError = Boom.wrap(err, err.statusCode);
-        // const wwwAuthHeader = get(err, 'body.error.header[WWW-Authenticate]');
-        // boomError.output.headers['WWW-Authenticate'] = wwwAuthHeader || 'Basic realm="Authorization Required"';
-
-        // throw boomError;
-      }
-
-      console.log('request failed yo', err);
-      throw err;
-    }
+    //return callAPI(this._noAuthClient, endpoint, clientParams, options);
   }
+
+  //callWithInternalUser = (endpoint, clientParams = {}, options = {}) => {
+  //  return callAPI(this._client, endpoint, clientParams, options);
+  //}
```

Comment:
Remove these commented out methods?

Reply:
I'm not sure, I don't have a good replacement for this whole thing yet. the `withCluster` that I removed could stay.